corehq/apps/hqadmin/views/users.py

```diff
@@ -197,7 +197,7 @@ def _parse_reports(xpath, xml_payload):
     @staticmethod
     def get_stats_from_xml(xml_payload):
         restore_id_element = xml_payload.find('{{{0}}}Sync/{{{0}}}restore_id'.format(SYNC_XMLNS))
-        restore_id = restore_id_element.text if restore_id_element else None
+        restore_id = restore_id_element.text if restore_id_element is not None else None
```

Comment:
one thought is that you should add a comment here explaining the situation so that some future reviewer doesn't suggest removing it.

Reply:
haha, fair point. Just added